src/sentry/utils/validators.py

```diff
@@ -11,6 +11,8 @@
 
 HEXADECIMAL_16_DIGITS = re.compile("^[0-9a-fA-F]{16}$")
 
+VALID_EMAIL_RE = re.compile(r"(^[a-zA-Z0-9*_.+-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$)")
```

Comment:
@JoshFerge is there a way to re-use django's email validator for this?

https://docs.djangoproject.com/en/4.2/ref/validators/#emailvalidator

Reply:
can use https://github.com/getsentry/sentry/blob/a412a627e77276c6eda43464bb1a6780984323dc/src/sentry/web/forms/fields.py#L73

Reply:
This looks to be an exclude list: https://github.com/getsentry/sentry/blob/a412a627e77276c6eda43464bb1a6780984323dc/src/sentry/utils/email/address.py#L60 and the regex here allows for wildcard searches

Reply:
IMO wildcard searches like `*@example.com` are nice to have in the search box

Reply:
ah right. yeah this regex makes sense then.

Reply:
Just realized this won't match email addresses in non standard English characters (unicode) (e.g: `garça@balão.com` won't match). But since this is just a convenience feature, it should be fine